### PR TITLE
Add new topic geodesy (picture and translations) db is ready

### PR DESCRIPTION
**Pull request by alcapat**
https://mf-chsdi3.dev.bgdi.ch/ltalp/shorten/7142a5e733

### CODE REVIEW

File: src/locales/fr.json [hunk 1]
```diff
@@ -169,6 +169,9 @@
 "funksender": "Emetteur radio",
 "funksender_service_link_href": "mailto:info@bakom.admin.ch",
 "funksender_service_link_label": "info@bakom.admin.ch",
+"geodesy": "Geodesia",
```
Comment (oterral) [inline src/locales/fr.json]: `Géodésie ` plutôt non ?

File: src/locales/fr.json [hunk 2]
```diff
@@ -169,6 +169,9 @@
...
+"geodesy_service_link_href": "https://www.swisstopo.admin.ch/fr/connaissances-faits/mensuration-geodesie.html",
+"geodesy_service_link_label": "Geodesia",
```
Comment (oterral) [inline src/locales/fr.json]: `Géodésie`

File: src/locales/fr.json [hunk 3]
```diff
@@ -413,6 +416,7 @@
 "topic_emapis_tooltip": "Améliorations structurelles",
 "topic_energie_tooltip": "Énergie",
 "topic_funksender_tooltip": "Emetteur radio",
+"topic_geodesy_tooltip": "Geodesia",
```
Comment (oterral) [inline src/locales/fr.json]: `Géodésie`

File: src/locales/it.json [hunk 4]
```diff
@@ -169,6 +169,9 @@
 "funksender": "Trasmettitore radio",
 "funksender_service_link_href": "mailto:info@bakom.admin.ch",
 "funksender_service_link_label": "info@bakom.admin.ch",
+"geodesy": "Informazioni sui fondi",
```
Comment (oterral) [inline src/locales/it.json]: No word in italian ?

File: src/components/topic/style/topic.less [hunk 5]
```diff
@@ -210,7 +210,10 @@
     background-position: -220px -660px;
   }
 
-  .ga-topics-sprite-geothermie  {
+  .ga-topics-sprite-geodesy {
+    background-position: -110px -780px;
+  }
+  .ga-topics-sprite-geothermie {
```
Comment (oterral) [inline src/components/topic/style/topic.less]: add a line please

File: src/locales/fr.json (hunk 1 above)
Comment (alcapat) [inline src/locales/fr.json]: I have sent an email to @mariokeusen in order to correct translations. 
Should be done when he come back of his holidays.

**Comment by alcapat** [comment]
French and italian translations are fxed

**Comment by alcapat** [comment]
Jenkins test

**Pull request merged**